### DISCUSSION TITLE
EmbedMolecule fail for a non-planar compound

### DISCUSSION THREAD

**Discussion by YoavShamir5** [rdkit/rdkit#7058] [Q&A] [answered]
Hello, 

I get an error when trying to run EmbedMolecule on a molecule of the following SMILES: "[H]C([H])=C([H])C(=O)N([H])C([H])([H])c1c([H])c([H])c([H])c(-c2nc(N([H])C34C([H])([H])C([H])(C3([H])[H])C4([H])[H])c3nc([H])nn3c2[H])c1[H]". I tried to useRandomCoords or to increase maxAttempts, but those did not help. I would appreciate any suggestions (in case the version is relevant, I am using 2022.09.5). Thanks!

![embed_issue](https://github.com/rdkit/rdkit/assets/39048833/45533000-067b-42a3-8c13-d9b373125b51)

**Comment by iuhgnor** [comment, marked as answer]
I tried to embed this mol using rdkit of 2022.9.05, and it worked. Maybe you can try the latest version.

```python
import rdkit
from rdkit import Chem
from rdkit.Chem import AllChem

print(rdkit.__version__)

smi = "[H]C([H])=C([H])C(=O)N([H])C([H])([H])c1c([H])c([H])c([H])c(-c2nc(N([H])C34C([H])([H])C([H])(C3([H])[H])C4([H])[H])c3nc([H])nn3c2[H])c1[H]"
mol = Chem.MolFromSmiles(smi)
mol = Chem.AddHs(mol)
AllChem.EmbedMolecule(mol)
mol.GetNumConformers()
```

Output:
```txt
2022.09.5
1
```

**Reply by YoavShamir5** [reply]
Thank you for the quick reply, this indeed was an issue on my side and not an RDKit-issue.
